articles/azure-maps/map-create.md

```diff
@@ -76,18 +83,18 @@ map.setCamera({
 });
 ```
 
-In the following code, a [Map object](/javascript/api/azure-maps-control/atlas.map) is created and the center and zoom options are set. Map properties, such as center and zoom level, are part of the [CameraOptions](/javascript/api/azure-maps-control/atlas.cameraoptions).
-
-<br/>
+Map properties, such as center and zoom level, are part of the [CameraOptions](/javascript/api/azure-maps-control/atlas.cameraoptions) properties.
```

Comment:
Do we need to also move `[CameraOptions]` link to bottom of the page?

Reply:
not as a part of the CodePen work.